doc/contributing/release-process.rst

```diff
@@ -188,7 +188,7 @@ Turn this file into a github issue with a checklist using this command::
 
    g. Run our script that checks for mistakes in the ckan.po files::
 
-        paster check-po-files ckan/i18n/*/LC_MESSAGES/ckan.po
+        ckan -c /path/to/ckan.ini check-po-files ckan/i18n/*/LC_MESSAGES/ckan.po
```

Comment:
You should test all these commands. The name of this one has changed. I've done a PR for it: https://github.com/ckan/ckan/pull/5191